### ISSUE TITLE
ImportError: No module named runner

### ISSUE THREAD

**Issue by suin** [fanout/pushpin#47234]
I'm trying pushpin with following [documentation](http://pushpin.org/docs/#running).
When I start pushpin with example config I get following error:

```console
$ git clone https://github.com/fanout/pushpin
$ cd pushpin/examples/config/
$ pushpin --verbose
Traceback (most recent call last):
  File "/usr/local/Cellar/pushpin/1.5.0/libexec/bin/pushpin", line 57, in <module>
    from runner import runner
ImportError: No module named runner
```

I installed via homebrew and puspin version is 1.5.0.

```console
$ brew install pushpin
$ pushpin --version
pushpin 1.5.0
```

Is any kind of  additional configuration required?

**Comment by jkarneges** [comment]
Pushpin looks for its runtime files based on the `libdir` property in pushpin.conf. If this property is missing or empty, then it tries to load the files relative to the current directory.

When you install via homebrew, a default configuration is saved in /usr/local/etc/pushpin/pushpin.conf with a correct libdir value. If you want to run a different configuration (e.g. the example config from the git repo, as it seems you are doing), then be sure to edit pushpin.conf and set the libdir value to the same as in the default homebrew configuration. This way Pushpin knows where to find its files.

**Comment by jkarneges** [comment]
I've decided that in a future version the executables should assume a hard-coded default for libdir based on the installation location, rather than relying on pushpin.conf to specify. I'll leave this issue open until this is resolved.

The reason is that the homebrew package sets libdir to a version specific location, and so if you upgrade pushpin then you'd have to edit pushpin.conf afterwards to point to the new libdir, which is weird.

**Comment by jkarneges** [comment]
Fixed in 70bf7e5300371700a72c6e553fb1c153f05f326b.
